docs/guide/api-javascript.md

```diff
@@ -134,9 +133,8 @@ async function build(
 **Example Usage:**
 
 ```js
-const path = require('path')
-const { build } = require('vite')
-
+import path from 'path'
+import { build } from 'vite'
 ;(async () => {
   await build({
     root: path.resolve(__dirname, './project'),
```

Comment:
We don't have `__dirname` here now. So we need to compute it in the example, no?

Reply:
Good point. will need to updat ethat.

Reply:
Updated with a polyfill since it's easier to understand than `filePathToUrl(new URL('foo', import.meta.url))`. At least until node supports `import.meta.resolve`